### DISCUSSION TITLE
can I search in Cyrillic?

### DISCUSSION THREAD

**Discussion by cactuslime1** [nroduit/Weasis#596] [Q&A]
Hi! I try to search my studies in Cyrillic and 0 results. Can i change to utf8 for fix this?

**Comment by nroduit** [comment]
Weasis works entirely with utf8. To investigate, I need a reproducible test case.
See also:
* https://github.com/nroduit/Weasis/issues/149
* https://github.com/nroduit/Weasis/issues/428
* https://github.com/nroduit/Weasis/issues/495
